src/trax/compiler/parser.ts

```diff
@@ -194,7 +194,8 @@ export function parse(src: string, filePath: string, options?: ParserOptions): (
                 }, skipProperty = false;
 
                 m.forEachChild((c) => {
-                    if (c.kind === SK.Identifier) {
+                    if (c.kind === SK.Identifier && !prop.name) {
+                        //First SK.Identifier is property name. Second one might be a variable as default value.
```

Comment:
trax changes should be submitted first on the trax repo (I will share it with you)